acceptance/apps_test.go

```diff
@@ -0,0 +1,52 @@
+package acceptance_test
+
+import (
+	"os"
+	"path"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Apps", func() {
+	var org = "apps-org"
+	BeforeEach(func() {
+		_, err := Carrier("create-org "+org, "")
+		Expect(err).ToNot(HaveOccurred())
+		_, err = Carrier("target "+org, "")
+		Expect(err).ToNot(HaveOccurred())
+	})
+	Describe("push and delete", func() {
```

Comment:
Future: Add `listing of pushed args`.

Reply:
Listing is already tested in this test to confirm that apps are pushed or deleted.

Reply:
Ah. I missed that. I believe because it was part of a larger block, not visually separate.